### PR TITLE
Jobs

### PR DESCRIPTION
**Pull request by DimitriPL5**
**DETAILS**
- Migrated the jobs page. 
- Connected to the DB to pull job data. 
- Added functionality for searching and filtering jobs. 

**IMPORTANT**
- After adding database env variable, run the following command to generate the prisma client: npx prisma generate

**To Do**
- Styling. 
- Allow for a search and filter to be done at the same time. 
- Clarify if group by companies will be on the new site (Additional work needed here if it will be included).

**TESTING**
- Done with 'npm run dev'.

### CODE REVIEW

File: src/interfaces/Search.interface.ts
Comment (styled-man) [inline src/interfaces/Search.interface.ts:1]: This is being imported in one file, can we just define it in there

File: src/app/jobs/JobModal.tsx
Comment (styled-man) [inline src/app/jobs/JobModal.tsx:1]: This file is not used anywhere, I think it safe to be deleted

File: src/Data/programData.ts
Comment (styled-man) [inline src/Data/programData.ts:1]: This file is not used anywhere, I think it safe to be deleted

File: src/Data/InitTeam.ts
Comment (styled-man) [inline src/Data/InitTeam.ts:1]: This file is not used anywhere, I think it safe to be deleted

File: src/Data/chapters.ts
Comment (styled-man) [inline src/Data/chapters.ts:1]: This file is not used anywhere, I think it safe to be deleted

File: src/Data/eventsData.ts
Comment (styled-man) [inline src/Data/eventsData.ts:1]: This file is not used anywhere, I think it safe to be deleted

File: src/Data/jobPostingsData.ts
Comment (styled-man) [inline src/Data/jobPostingsData.ts:1]: This file is not used anywhere, I think it safe to be deleted

File: src/app/api/jobRoutes.ts
Comment (styled-man) [inline src/app/api/jobRoutes.ts:1]: I don't think this file has any functionality

File: src/app/api/[trpc].ts
Comment (styled-man) [inline src/app/api/[trpc].ts:1]: I don't think this file has any functionality

File: src/app/api/search.ts
Comment (styled-man) [inline src/app/api/search.ts:1]: I don't think this file has any functionality

File: src/Data/programData.ts
Comment (styled-man) [inline src/Data/programData.ts:1]: I don't think this file has any functionality

File: src/Data/page.tsx
Comment (styled-man) [inline src/Data/page.tsx:1]: I don't think this file has any functionality

File: src/app/jobs/DeleteModal.tsx
Comment (styled-man) [inline src/app/jobs/DeleteModal.tsx:1]: I don't think this file has any functionality

File: src/app/jobs/JobModal.tsx
Comment (styled-man) [inline src/app/jobs/JobModal.tsx:1]: I don't think this file has any functionality

**Review by styled-man — CHANGES_REQUESTED** [review]

**Review by styled-man — APPROVED** [review]

**Review by styled-man — APPROVED** [review]

**Review by raidel-a — DISMISSED** [review]
Change events_tracker directory

**Review by raidel-a — APPROVED** [review]
lgtm

**Review by styled-man — APPROVED** [review]

**Pull request merged**